src/routes/cache-hooks/cache-hooks.controller.spec.ts

```diff
@@ -768,4 +768,54 @@ describe('Post Hook Events (Unit)', () => {
       webHookExecutionDelayMs,
     );
   });
+
+  it.each([
+    {
+      type: 'MODULE_TRANSACTION',
+      module: faker.finance.ethereumAddress(),
+      txHash: faker.string.hexadecimal({ length: 32 }),
+    },
+  ])('$type clears Safes by module', async (payload) => {
+    const chainId = faker.string.numeric();
+    const safeAddress = faker.finance.ethereumAddress();
+    const moduleAddress = faker.finance.ethereumAddress();
+    const safesByModule = {
+      safes: [
+        faker.finance.ethereumAddress(),
+        faker.finance.ethereumAddress(),
+        faker.finance.ethereumAddress(),
+      ],
+    };
+    const cacheDir = new CacheDir(
+      `${chainId}_modules_${moduleAddress}`,
+      faker.string.alpha(),
+    );
+    await fakeCacheService.set(cacheDir, faker.string.alpha());
+    const data = {
+      address: safeAddress,
+      chainId: chainId,
+      ...payload,
+    };
+    networkService.get.mockImplementation((url) => {
```

Comment:
Why is this network call mock needed? The cache key is already populated by `fakeCacheService.set` in the line 793, right?

Reply:
It's a leftover from using another test as a reference 🙈 I've removed it in cbc68a8b4ec8699a56ccdd1a95c5226325bec844.